src/api/lib/tasks/dev/requests.rake

```diff
@@ -199,5 +199,333 @@ namespace :dev do
 
       puts "* Request with delete action #{request.number} has been created."
     end
+
+    desc 'Copy last 10 requests from openSUSE:Factory'
+    task requests_from_open_suse_factory: :development_environment do
+      require 'factory_bot'
+      include FactoryBot::Syntax::Methods
+
+      users = []
```

Comment:
I don't like that you pass this "global" variable around 3 levels deep. Why are you doing this?

Reply:
It's removed. Not needed anymore